broker/cloud_functions/lsst/ps_to_storage/deploy.sh

```diff
@@ -9,11 +9,12 @@ testid="${1:-test}"
 teardown="${2:-False}"
 # name of the survey this broker instance will ingest
 survey="${3:-lsst}"
-# schema version
-versiontag="${4:-v7_3}"
-region="${5:-us-central1}"
+region="${4:-us-central1}"
 PROJECT_ID=$GOOGLE_CLOUD_PROJECT # get the environment variable
 
+MODULE_NAME="to-storage"  # lower case required by cloud run
```

Comment:
```suggestion
MODULE_NAME="alerts_to_storage"  # lower case required by cloud run
```

Our standard syntax is to use an underscore to separate the words in the base name of a module. The survey name then gets prepended with a dash. That way, it is easy to write code that parses them. I also suggest to add "alerts" or something similar to the name, since this is not the only module that will send data to storage.

Reply:
"alerts" added to the name! 

Underscores are not permitted when naming Cloud Run services. I am having trouble finding where it is explicitly stated in Google's documentation, but I can confirm this is true through personal experience. The following error appears when underscores are used: 
`The name must use only lowercase alphanumeric characters and dashes, cannot begin or end with a dash, and cannot be longer than 63 characters.`

Reply:
Ah, ok. Nothing we can do about that then.